bin/compile

```diff
@@ -25,20 +25,23 @@ function export_env_dir() {
 function indent() {
   sed -u 's/^/       /'
 }
-
 BUILD_DIR=${1:-}
 CACHE_DIR=${2:-}
 
 export_env_dir "$3"
 
 # Create the cache directory if not exist.
 mkdir -p $CACHE_DIR
-
-cd $BUILD_DIR
+if [ -n "$FLUTTER_BUILD_DIR" ]; then
+  APP_DIR=$BUILD_DIR/$FLUTTER_BUILD_DIR
```

Comment:
`APP_DIR` może też dezorientować, bo na Heroku to wszystko dzieje się w katalogu `/app`. To tak naprawdę jest `FLUTTER_SOURCE_DIR` tylko w wersji ścieżki absolutnej (z dodanym `$BUILD_DIR`). Może powinno się nazywać po prostu `SOURCE_DIR`?